sdk/scheduler/src/main/java/com/mesosphere/sdk/dcos/secrets/Secret.java

```diff
@@ -0,0 +1,92 @@
+package com.mesosphere.sdk.dcos.secrets;
+
+import java.util.Collection;
+
+/**
+ * Secret class represents an HTTP API model for creating and updating a secret.
+ */
+public class Secret {
```

Comment:
I was originally using the `JSONObject` but now I've switched to the `ObjectMapper`.